databricks_cli/pipelines/api.py

```diff
@@ -146,12 +182,22 @@ def convert_from_libraries(cls, libraries):
 
     @classmethod
     def convert_to_libraries(cls, lib_objects):
+        """
+        Deserialize LibraryObjects
+        :param lib_objects: List[LibraryObject]
+        :return: List[Dictionary{String, String}]
+        """
         libraries = []
         for lib_object in lib_objects:
             libraries.append({lib_object.lib_type: lib_object.path})
         return libraries
 
     def __eq__(self, other):
+        """
+        Equality Comparison
+        :param other:
+        :return:
+        """
```

Comment:
It's probably good to document all the stuff if we're going to include this. For this one, I think it's pretty obvious what it does and so the doc can be removed.